docs/usage/opentelemetry.md

```diff
@@ -0,0 +1,16 @@
+---
+title: OpenTelemetry
+description: How to use OpenTelemetry with Renovate
+---
+
+# OpenTelemetry and Renovate
+
+Renovate supports OpenTelemetry which is an emerging monitoring standard.
+
+To activate the instrumentation, the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` has to be set.
+This sets the endpoint where to send the telemetry data. If this endpoint is set, all other environment variables defined
+by the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md)
+are supported.
```

Comment:
Lines in Markdown files in this projects contain an entire single sentence, even if that means the line is 500+ chars long, like here https://github.com/renovatebot/renovate/blob/97fd4b884c4989cdac61bd9feb13e727d16e7e11/docs/usage/configuration-options.md#L2769

So:
```suggestion
This sets the endpoint where to send the telemetry data. If this endpoint is set, all other environment variables defined by the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md) are supported.
```